### PR TITLE
Experiments with bubbles

### PR DESCRIPTION
**Pull request by chaoran-chen**
Hey. After the great presentation of D3 (thanks @MusicConnectionMachine/group-4!) and getting inspirations from the meeting with @gyachdav and @vviro, I've played around with D3 and built a small visualization.

The data is from [here](https://github.com/MusicConnectionMachine/RelationshipsG3/issues/21). It has some animations - so give it a try: just pull the repository, go into `experiments/relations_exploring_graph/`, run `yarn install` and start an HTTP-server.

I know that we have decided in that meeting, that we will only use a table at the beginning, but maybe this visualization could be still useful someday.

P.S. Concerning the cyan bubble in the first image, blame @MusicConnectionMachine/group-3, not me! ;-)

### CODE REVIEW

**Comment by martomi** [comment]
I think you meant `yarn install` since there's no bower configuration file? I've tried starting a simple HTTP python server, but it doesn't work on my machine. How do you do it?

File: experiments/relations_exploring_graph/main.js [hunk 1]
```diff
@@ -0,0 +1,287 @@
...
+        i = (i + 1) % positions.length;
+      }
+    })();
+  }
+
+
+  _createTooltips() {
+    // TODO Delete old tooltips
+    this.nodes.forEach((node) => {
+      console.log(document.getElementById('bubble-' + node.id));
```
Comment (martomi) [inline experiments/relations_exploring_graph/main.js]: Remove that console.log

**Review by martomi — APPROVED** [review]
I managed to run it with Chrome, it doesn't work on Safari because I guess `fetch` is not supported there. Approving since it's just an experiment at this stage. Looks good otherwise 👍 .

**Comment by chaoran-chen** [comment]
Oh, yes, of course, I meant yarn. Are you getting any error messages? I've used the [node http-server](https://www.npmjs.com/package/http-server)

**Pull request merged**